tests/test-All_in_One_SEO_Pack-get_main_description.php

```diff
@@ -0,0 +1,267 @@
+<?php
```

Comment:
@EkoJr can you sync with master and also put this in a relevant folder, maybe `general`? Also, the class name could be less verbose :)

Some comments from https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/1588 hold true here as well.

Reply:
I ran into the same problem with setup_post. I needed the post IDs, and I also needed to post with an excerpt, and without an excerpt.

With the `i-1518-r*` (rebased) branches, I created (proposed) method in the base class to allow more flexibility, and still does the assert checks; also checks WP_Post variables.
https://github.com/EkoJr/all-in-one-seo-pack/commit/d29c0380756c978e287eb2a7d3c0291428d3efec#diff-2119825f6f43fe8b6faf6b06d2d10858R179